tests/suit/forklift_test.go

```diff
@@ -84,6 +88,7 @@ var _ = FDescribe("Forklift", func() {
 			By("Verify created plan")
 			Expect(plan).ToNot(BeNil())
 			Expect(plan.Spec.TransferNetwork).To(BeNil())
+			Expect(plan.Annotations[AnnPopulatorLabels]).To(Equal("True"))
```

Comment:
as it's unrelated to the transfer network, how about adding another case for this?